mods/ts/sequences/civilian.yaml

```diff
@@ -368,17 +389,22 @@ capyr03:
 ^city2x3:
 	Inherits: ^cabase
 	Defaults:
-		Offset: 12, -30, 30
+		Offset: 12, -30, 36
```

Comment:
The depth sprite is probably not centered relative to the artwork, try moving that to the right instead.  If ^ is the case then adding a uniform offset to Z may indeed fix the right hand side, but makes the imbalance on the left even larger, causing potential clipping problems there instead.